### ISSUE TITLE
UncaughtException when running Markbind Serve in non-Markbind directories.

### ISSUE THREAD

**Issue by gerteck** [MarkBind/markbind#2570]
### Please confirm that you have searched existing issues in the repo

Yes, I have searched the existing issues

### Any related issues?

#1715

### Tell us about your environment

Win 10

### MarkBind version

3.1.1 - 5.5.2

### Describe the bug and the steps to reproduce it

When markbind serve is run in stray directories (non-MarkBind configured folders), there is an uncaught exception that should be caught, or at least not shown to the user, to avoid confusion. (unless this is intentional).

> `The "path" argument must be of type string.`

v3.1.1:
![image](https://github.com/user-attachments/assets/24182eb1-88c0-4244-8c2d-370b186b54ea)

v5.5.2:
![image](https://github.com/user-attachments/assets/e5beebdf-6628-42d4-9b06-eace16bbdf3d)

Minor nit to improve CLI experience.

### Expected behavior

Error not shown.

**Comment by tlylt** [comment]
Yes, I think https://github.com/MarkBind/markbind/issues/1825 is relevant to this issue. I guess it is a known behavior that markbind serve does not work in non-markbind directories, but improving the error handling/message is indeed good to have.
